File: WeChatAuto4_X/WeChatAuto/Models/ChatSimpleMessage.cs
using System;
using System.Globalization;
using System.Text.Json.Serialization;
using Newtonsoft.Json;

namespace WeChatAuto.Models
{
    public class ChatSimpleMessage
    {
        /// <summary>
        /// 微信名称
        /// </summary>
        [JsonProperty("who")]
        public string Who { get; set; }
        /// <summary>
        /// 消息
        /// </summary>
        [JsonProperty("message")]
        public string Message { get; set; }
        /// <summary>
        /// 消息日期
        /// </summary>
        [JsonProperty("send_date_time")]
        public string SendDateTime { get; set; }
        /// <summary>
        /// 消息日期,日期时间格式
        /// </summary>
        [JsonProperty("date_time")]
        public DateTime DateTime { get; set; }

        /// <summary>
        /// 唯一字符串
        /// </summary>
        [JsonProperty("unique_string")]
        public string UniqueString { get; set; }

        public override string ToString()
        {
            return JsonConvert.SerializeObject(this);
        }
    }
}

File: WeChatAuto4_X/wechatauto_py/src/wechat_auto_sdk/models/chat_simple_message.py
from pydantic import BaseModel
from datetime import datetime


class ChatSimpleMessage(BaseModel):
    # 微信名称
    who: str
    # 消息
    message: str
    # 消息日期,字符串格式
    send_date_time: str
    # 消息日期,日期时间格式
    date_time: datetime
    # 唯一字符串
    unique_string: str
